php/commands/sites.php

```diff
@@ -341,8 +341,8 @@ public function mass_update($args, $assoc_args) {
         $update = $noupdatedb ? false : true;
         $xoption = Input::optional($assoc_args, 'xoption', 'theirs');
         if (!$report) {
-          $confirmed = Input::yesno("Apply upstream updatefs to %s ( run update.php:%s, xoption:%s ) ", array($site->getName(), var_export($update,1), var_export($xoption,1)));
-          if( !$confirmed ) continue; // Suer says No, go back to start.
+          $confirmed = Input::yesno("Apply upstream updates to %s ( run update.php:%s, xoption:%s ) ", array($site->getName(), var_export($update,1), var_export($xoption,1)));
+          if( !$confirmed ) continue; // User says No, go back to start.
```

Comment:
Style guide also wants no spaces after parentheses, e.g.
`if(!$confirmed) continue; // User says No, go back to start.`